@navikt/core/tokens/figma/tests/deepen.test.ts

```diff
@@ -0,0 +1,73 @@
+import deepen from "../deepenJson";
+
+describe("Check deepen parsing of key name", () => {
+  test("Convert spacing", () => {
+    const before = {
+      "navds-spacing-1": { value: "0.25rem" },
+      "navds-spacing-2": { value: "0.50rem" },
+      "navds-spacing-3": { value: "0.75rem" },
+      "navds-spacing-4": { value: "1rem" },
+    };
+
+    const after = {
+      navds: {
+        spacing: {
+          "1": { value: "0.25rem" },
+          "2": { value: "0.50rem" },
+          "3": { value: "0.75rem" },
+          "4": { value: "1rem" },
+        },
+      },
+    };
+    expect(JSON.stringify(deepen(before))).toBe(JSON.stringify(after));
+  });
+
+  test("Convert colors", () => {
```

Comment:
Disse testene har litt mye data for min smak. Kan du redusere de uten å fjerne noe av kompleksiteten i testen?

Reply:
F.eks. til "red: 100 + green: 100, 200 + semantic: border, background"